Lib/fontTools/ttLib/tables/_p_o_s_t.py

```diff
@@ -85,7 +87,8 @@ def decode_format_2_0(self, data, ttFont):
 		indices.frombytes(data[:2*numGlyphs])
 		if sys.byteorder != "big": indices.byteswap()
 		data = data[2*numGlyphs:]
-		self.extraNames = extraNames = unpackPStrings(data)
+		maxIndex = max(indices[:numGlyphs])
```

Comment:
I think`max(indices)` would do the same here, because the `indices` local variable is an array of USHORT that is unpacked from a bytes string (`data`) which is maximum `2*numGlyphs` long (i.e. `indices.frombytes(data[:2*numGlyphs])`)

Reply:
Fixed.